recipes/dask-cuda/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "dask-cuda" %}
+{% set version = "0.18.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dask-cuda-{{ version }}.tar.gz
+  sha256: f5127667bbb6730e23e22016fb1a64f308d8e8455fa61993990e5d237df75396
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - dask-cuda-worker = dask_cuda.cli.dask_cuda_worker:go
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
```

Comment:
We moved to Python 3.7 as a minimum here ( https://github.com/rapidsai/dask-cuda/pull/345 ). Though this may still work on Python 3.6, we just don't test it any more

Reply:
Trying to clarify this further with PR ( https://github.com/rapidsai/dask-cuda/pull/576 ). If there's anything else we can do to clear this up, please let us know 🙂

Reply:
I'm OK just using >=3.7. We should drop 3.6 anytime now...